src/main/java/io/sc3/plethora/integration/computercraft/meta/item/MediaItemMeta.kt

```diff
@@ -1,12 +1,16 @@
 package io.sc3.plethora.integration.computercraft.meta.item
 
-import dan200.computercraft.impl.MediaProviders
+import dan200.computercraft.api.media.IMedia
+import dan200.computercraft.api.media.MediaLookup
+import dan200.computercraft.api.media.MediaProvider
+import dan200.computercraft.shared.media.items.RecordMedia
 import io.sc3.plethora.api.meta.BasicMetaProvider
 import net.minecraft.item.ItemStack
 
 class MediaItemMeta : BasicMetaProvider<ItemStack>() {
   override fun getMeta(target: ItemStack): Map<String, *> {
-    val media = MediaProviders.get(target) ?: return emptyMap<String, Any>()
+
+    val media = if(target.item is IMedia) target.item as IMedia else RecordMedia.INSTANCE
```

Comment:
Think you can use `dan200.computercraft.api.media.MediaLookup` here instead:

```suggestion
    val media = MediaLookup.get().find(target, null)
```

Reply:
With this change There's a possibility that the `media` variable could be null.

Would it be preferred to wrap in a null check and then return an empty map if there's no metadata, or use nullable field access to provide the current map but each item set to null?

Reply:
You can probably just do this:

```
val media = MediaLookup.get().find(target, null) ?: return emptyMap()
```

Reply:
This is good, just had to give it the types.
Thanks :)